parlai/tasks/interactive/worlds.py

```diff
@@ -16,4 +18,57 @@ class InteractiveWorld(DialogPartnerWorld):
     especially for those cases for given tasks.
     """
 
-    pass
+    def __init__(self, opt, agents, shared=None):
+        super().__init__(opt, agents, shared)
+        self.init_contexts()
+        self.turn_cnt = 0
+
+    def init_contexts(self):
+        """
+        Override to load or instantiate contexts to be used to seed the chat.
+        """
+        pass
+
+    def get_contexts(self):
+        """
+        Override to return a pair of contexts with which to seed the episode.
+
+        This function will be called before the first turn of every episode.
+        """
+        return ['__SILENCE__', '']
+
+    def parley(self):
+        """
+        Agent 0 goes first.
+
+        Alternate between the two agents.
+        """
+        if self.turn_cnt == 0:
+            self.p1, self.p2 = self.get_contexts()
+
+        acts = self.acts
+        agents = self.agents
+        if self.turn_cnt == 0:
+            # add the context on to the first message to agent 0
+            act = {}
+            act['text'] = self.p1
+            act['episode_done'] = False
+            act['id'] = 'context'
```

Comment:
do we always want to add a contexT? i'm not sure about this 

cc @stephenroller

Reply:
hmm, i guess i could just add if it isn't an empty string..

Reply:
Does it make more sense to have get_context return something other than an empty string for that? E.g. it returns None instead?

This is fine i think, but maybe the semantics could be improved.